### PR TITLE
Remove duplication from packages

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: tools/generate-package-lists [hunk 1]
```diff
@@ -70,6 +70,7 @@ def main():
             pkgs = [ {"name": p["name"], "summary": p["summary"]} for p in json.loads(out)['packages'] ]
             filtered = []
             [ filtered.append(p) for p in pkgs if p["name"] not in list(map(lambda p: p["name"], filtered)) ]
+            filtered.sort(key=lambda p: p["name"].lower())
```
Comment (Gundersanne) [inline tools/generate-package-lists:73]: I know they're all lowercase, but to be triple sure :)

**Comment by teg** [comment]
Looking good, but does this break backwards compatibility in any way? Will we fail if a caller passes a version field?

**Comment by Gundersanne** [comment]
> Looking good, but does this break backwards compatibility in any way? Will we fail if a caller passes a version field?

We never required a version field to specify packages, just the package name.

This breaks compatibility a little if the caller depended on the version field being present when querying /packages, none of the current users (IBFE and edge) do.  And honestly i've kind of given up on backwards compatibility until we reach public beta .-.

**Review by teg — APPROVED** [review]
If you checked the only two real users, then I think we are good

**Pull request merged**